expenses/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+
+class Expenses(models.Model):
+    # house_id = models.ForeignKey('user_id', on_delete=models.CASCADE)
+    amount = models.IntegerField()
+    year = models.IntegerField()
+
+    class Category(models.TextChoices):
+        RENT = 'Rent'
+        MORTGAGE = 'Mortgage'
+        BILLS = 'bills'
+        TRANSPORTATION = 'Transportation'
+        CLOTHING = 'Clothing'
+        HEALTHCARE = 'Healthcare'
+        FOOD = 'Food'
+        INSURANCE = 'Insurance'
+        KIDS = 'Kids'
+        CULTURE = 'Culture'
+        VACATIONS = 'Vacations'
+        OTHER = 'Other'
+
+    category = models.CharField(
+        max_length=32,
+        choices=Category.choices,
+        default=Category.OTHER
+    )
+
+    class Monthes(models.TextChoices):
```

Comment:
Months*

Reply:
a quick fix is ctrl + f, type Monthes
click on the line on the left of the opened window
and replace all to: Months
(in VScode at least)

Reply:
I fixed it